lib/ansible/modules/remote_management/hpilo/hpilo_boot.py

```diff
@@ -65,12 +65,6 @@
     - As a safeguard, without force, hpilo_boot will refuse to reboot a server that is already running.
     default: no
     choices: [ "yes", "no" ]
-  ssl_version:
```

Comment:
Removing a parameter that people are using will break existing playbooks that depend on it.
This is a definite NO-GO.